tests/test_wrappers/test_sklearn_wrapper.py

```diff
@@ -549,3 +549,25 @@ def test_get_feature_names_out_ohe(varlist, df_vartypes):
         ]
 
         assert output_feat == transformer.get_feature_names_out(varlist)
+
+
+def test_sklearn_transformer_wrapper():
```

Comment:
I would rename this test for something with more information about what the aim of the test is. 

In this particular case, we are wrapping the FunctionTransformer and we want to ensure that this transformer can work with categorical variables.  So maybe rename to `test_function_transformer_works_with_categoricals`.

We could also add a test to test that it work with numericals (I am getting greedy, sorry, but let me make a code suggestion for the new test):

```

    X = pd.DataFrame({
        "col1": [1, 2, 3],
        "col2": ["a", "b", "c"]
        })

    X_expected = pd.DataFrame({
        "col1": [2, 3, 4],
        "col2": ["a", "b", "c"]
        })

    transformer = SklearnTransformerWrapper(
        FunctionTransformer(lambda x: x+1,
        variables=["col1"]
    )
```

you get the gist ;)

Reply:
Hi there, I have included these changes and tested the functions. Everything seems ok now, would you please let me now if there's anything else I can do to make it better?

Reply:
@solegalli, it turns out I accidentally included this file in formatting changes as well... do you reckon we could keep it and merge it together with the others or should I remove it from this PR?
`tests/test_selection/test_check_estimator_selectors.py` 

Thanks a bunch